Test_oM/UnitTests/TestData.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*	
+ * This file is part of the Buildings and Habitats object Model (BHoM)	
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.	
+ *	
+ * Each contributor holds copyright over their respective contributions.	
+ * The project versioning (Git) records all such contribution source information.	
+ *                                           	
+ *                                                                              	
+ * The BHoM is free software: you can redistribute it and/or modify         	
+ * it under the terms of the GNU Lesser General Public License as published by  	
+ * the Free Software Foundation, either version 3.0 of the License, or          	
+ * (at your option) any later version.                                          	
+ *                                                                              	
+ * The BHoM is distributed in the hope that it will be useful,              	
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               	
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 	
+ * GNU Lesser General Public License for more details.                          	
+ *                                                                            	
+ * You should have received a copy of the GNU Lesser General Public License     	
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      	
+ */
+
+using BH.oM.Base;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+
+namespace BH.oM.Test.UnitTests
+{
+    [Description("Defines input data and expected output data for a particular unit test.")]
+    public class TestData : BHoMObject
```

Comment:
IImmutable?! 
Sorry to ask 😄

Reply:
I would actually agree in this case. Inputs and Outputs are not orthogonal to each other

Reply:
done. Ofc impossible to amke things 100% immutable, as you can always modify the objects in the collection, but if you start hacking it that much, we kind of have to trust the the person doing that knows what they are doing! 😄